source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.yml

```diff
@@ -0,0 +1,79 @@
+# Test SDAM error handling.
+runOn:
+    # failCommand appName requirements
+    - minServerVersion: "4.4"
+
+database_name: &database_name "sdam-tests"
+collection_name: &collection_name "sdam-minPoolSize-error"
+
+data: []
+
+tests:
+  - description: Command error on minPoolSize background creation
+    # Configure the initial monitor handshake to succeed but the
+    # first or second background minPoolSize establishments to fail.
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { skip: 2 }
```

Comment:
I think this should be `skip: 3` to account for the initial monitor handshake, the initial streaming hello, and the initial RTT connection attempt.

Reply:
fixed